curvefs/src/mds/mds.cpp

```diff
@@ -220,5 +228,34 @@ void Mds::InitLeaderElection(const LeaderElectionOptions& option) {
     leaderElection_ = std::make_shared<LeaderElection>(option);
 }
 
+void Mds::InitChunkIdAllocator() {
+    std::shared_ptr<EtcdClientImp> etcdClient =
+        std::make_shared<EtcdClientImp>();
+    int res = etcdClient->Init(options_.etcdConf, options_.etcdClientTimeout,
```

Comment:
etcdclient is duplicated, use https://github.com/opencurve/curve/blob/50fb805063da9bfb6bef3add0d6295412426ae7a/curvefs/src/mds/mds.h#L92 directly

Reply:
fix